react-components/table-intro/step5.md

```diff
@@ -1,8 +1,8 @@
-A typical case for the table component is the need to handle large datasets elegantly. This is addressed on both the styling and interactivity side. In this step, styling techniques are applied to improve the experience.
+Table components are commonly used to handle large datasets. This is addressed on both the styling and interactivity side. In this step, apply styling techniques to improve the experience.
```

Comment:
I think styling techniques is a little broad here

Reply:
I revised this section also, think it's clearer now. Let me know what you think!